### ISSUE TITLE
Fix header accessibility violations

### ISSUE THREAD

**Issue by alizedebray** [swisspost/design-system#4862]
Axe detects accessibility violation in the header.

**Comment by gfellerph** [comment]
@alizedebray please add infos about the reports.

**Comment by alionazherdetska** [comment]
**Axe dev tools:**

![Image](https://github.com/user-attachments/assets/ae8ea2cc-cf8e-43a9-a191-8c784a83fc58)

**ARC Toolkit:**

![Image](https://github.com/user-attachments/assets/0206b2dd-42b6-467d-b0f0-3840258cc008)
